Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js

```diff
@@ -922,10 +922,28 @@ WI.DOMNodeStyles = class DOMNodeStyles extends WI.Object
                             property.overridingProperty = effectiveProperty;
                             continue;
                         }
-                    } else if (effectiveProperty.important || !property.important || effectiveProperty.ownerStyle.node !== property.ownerStyle.node) {
-                        property.overridden = true;
-                        property.overridingProperty = effectiveProperty;
-                        continue;
+                    } else {
+                        let propertyIsLessImportant = property.ownerStyle.isStartingStyle;
+                        let effectivePropertyIsLessImportant = effectiveProperty.ownerStyle.isStartingStyle;
```

Comment:
Maybe this is just me because I understand how `@starting-style` works, but I'd name this `(effective)propertyIsStartingStyle` because it would make the code below make a bit more sense when you read it.

Even if the current name was made more generic to account for future at-rules, it's not guaranteed future at-rules re-use this exact same logic (and it you might need to define the importance of different at-rules relative to each other), so I'd rather be explicit with this being about `@starting-style`